Dockerfile

```diff
@@ -20,12 +26,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone &
     vim \
     clang-tidy-14 \
     clang-format-14 \
-    libpng-dev && \
-    # Add source for practicerom-dev install
-    curl https://practicerom.com/public/packages/debian/pgp.pub | apt-key add - && \ 
-    echo deb http://practicerom.com/public/packages/debian staging main >/etc/apt/sources.list.d/practicerom.list && \
-    apt-get update && \
-    apt-get install -y \
+    libpng-dev \
     practicerom-dev && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
```

Comment:
I would say we should split these out as well so this just becomes a list of the packages we are installing.